### PR TITLE
Adds support for content templates

### PR DESCRIPTION
**Pull request by skttl**
#101 

This is based on the work done in #100

Uses the contentResource to get the blueprint scaffold. Besides that, it's more or less just like the implementation in Inner Content.

I upgraded to Umbraco 7.7, because that is where blueprints are added.

See it in action here: https://i.imgur.com/juKHnfr.gifv (sorry for the slow db connection :))

### CODE REVIEW

**Comment by leekelleher** [comment]
@skttl - Thanks for all the PRs, we're loving them! #h5yr

With this one, I'm going to merge it in to a new "v1.0" branch (that I've just created). As I'm not sure bumping up the minimum Umbraco version dependency on a patch release is a good move (in terms of package support).

I'm thinking that if we could get a v0.6.0 release out, (that includes your other PRs), then we an lead up to (finally) getting a v1.0 release out.  Which, I'm happy to bump the Umbraco version with.

**TL;DR; I'm merging in now, but it wont be released just yet.**
